codespell_lib/data/dictionary.txt

```diff
@@ -19389,8 +19436,14 @@ indepently->independently
 inderect->indirect
 inderts->inserts
 indes->index
+indescriminent->indiscriminate
 indespensable->indispensable
 indespensible->indispensable
+indever->endeavor
```

Comment:
Don't forget British spelling `endeavour`.

Add `endeavour` to GB_US dictionary?

Reply:
> Don't forget British spelling `endeavour`.

Added.

> Add `endeavour` to GB_US dictionary?

It's already there!